backend/src/nodes/nodes/utility/convert_number.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import NumberInput, TextInput
+from ...properties.outputs import NumberOutput
+from . import category as UtilityCategory
+
+
+@NodeFactory.register("chainner:utility:convert_number")
+class TextLengthNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Converts a text to base-10."
+        self.inputs = [
+            TextInput("Text", min_length=0),
+            NumberInput("Base", default=10, minimum=0, maximum=64),
+        ]
+        self.outputs = [
+            NumberOutput("Value"),
```

Comment:
While we currently don't have a way to parse numbers in the type system, we could at least declare that we're only returning ints.
```suggestion
            NumberOutput("Value", output_type="int"),
```